src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,25 @@
+fun main(args: Array<String>) {
+    scenario(12,8)
+}
+
+fun scenario(numberOfCars : Int, capacity : Int) {
+    /*
+    * Creates a given a number of cars, every other car wants to drive
+    * A road network is created with the given capacity
+    * The road network is analyzed and it is printed whether a car is delayed or not
+    */
+    val listOfCars = ListOfCars()
```

Comment:
But didn't you suggest du create a new class, that creates the list of cars? Or did I misunderstand?